### ISSUE TITLE
Use Weathericons with Nerd Fonts

### ISSUE THREAD

**Issue by matt70tx** [chubin/wttr.in#363]
Nerd Fonts, https://www.nerdfonts.com/cheat-sheet , has integrated weather icons glyphs from https://github.com/erikflowers/weather-icons project.

With the use of format=v2 (or v3), would it be possible to add an option to use these instead of emojis? On my Windows Terminal, some emojis are correct size and others not. On my work machine with Windows 7, some emojis are missing.

**Comment by chubin** [comment]
Yes, sure, it is a very cool idea! We can add a new view, say `v2n` (n for `nerd`) that is supposed to be used with this view. Are you ready to make a pull-request for that? I will make initial preparations, and you will add the icons into the v2n view file. What do you say?

**Comment by moonwitch** [comment]
If this is still applicable, I'd be happy to try and help?

**Comment by chubin** [comment]
@moonwitch Yes Kelly it would be great! Please help us.
You can simply add the correct symbols (as a new dictionary)
to https://github.com/chubin/wttr.in/blob/master/lib/constants.py
for the beginning. And then we will try to add an additional view
to use this symbols

**Comment by Gnostiphage** [comment]
@chubin @moonwitch I was coincidentally searching for a solution to this today and happened across this conversation. I have some spare time right now and would like to contribute as well, plus I know the weather-fonts have more options for day/night weather, moon phases for each day in the cycle, and more in the way of wind direction and wind scale. I wouldn't know where to go about implementing that in your code, but I can at least add those symbols as new separate dictionaries.

**Comment by chubin** [comment]
I've added initial support for nerd fonts: please take a look at the `v2d`/`v2n` views:

```
curl v2d.wttr.in
```

I don't have currently a terminal with nerdfonts at hand (why actually?),
can you please post a screenshot?
What is missing?

**Comment by moonwitch** [comment]
It seems to work fine, other then some alignment issue - however, that could easily be down to the terminal. 
I use Hasklug Nerd Font. I'll give it a go with a few different nerd fonts.  But so far, it looks ok. 
![2020-06-07-18:29:46](https://user-images.githubusercontent.com/15063/83974339-e7a01e00-a8ec-11ea-8e68-da3e7b3c6dbe.png)

**Comment by moonwitch** [comment]
Same in Tilix; Hasklug Nerd Font as font. I do notice a good difference between the fonts. So I think I'll best test with Tilix. 

![2020-06-07-18:35:16](https://user-images.githubusercontent.com/15063/83974467-c55ad000-a8ed-11ea-8095-a4f71b61be62.png)

**Comment by chubin** [comment]
@moonwitch thank you for testing; please test it again; it should look better now

**Comment by moonwitch** [comment]
![2020-06-07-20:32:03](https://user-images.githubusercontent.com/15063/83977022-1ffc2800-a8fe-11ea-8a39-d9909980c9c6.png)

Seems ok to me :)

**Comment by chubin** [comment]
Thank you! Now we have initial supportof Nerd Fonts in the v2 view,
What is still not implemented: (1) one-line output (2) Nerd Fonts support in HTML/PNG
(open https://v2d.wttr.in in browser to see it)

**Comment by miikanissi** [comment]
> ## Todo
> 
>     * [x]  v2 support
> 
>     * [ ]  v2 png support
> 
>     * [ ]  one-line format support
> 
> 
> ## Original issue
> 
> Nerd Fonts, https://www.nerdfonts.com/cheat-sheet , has integrated weather icons glyphs from https://github.com/erikflowers/weather-icons project.
> 
> With the use of format=v2 (or v3), would it be possible to add an option to use these instead of emojis? On my Windows Terminal, some emojis are correct size and others not. On my work machine with Windows 7, some emojis are missing.

Are there plans to get back to this feature request? Especially the one-line format would be helpful to have.

**Comment by chubin** [comment]
You mean the nerd-fonts support? Sure

Have you tested the v2 view with nerd fonts?
```
curl v2d.wttr.in
curl v2t.wttr.in
```

Do you like it?

**Comment by miikanissi** [comment]
Yeah I like it a lot, the v2 interface is great as well. I was mainly asking about the one-line format as it's a great way to curl weather for a status bar for example!

**Comment by chubin** [comment]
Yes; you are talking only about the nerd fonts? Because you know that wttr.in supports one-line format per-se already, right?

**Comment by miikanissi** [comment]
Yes only about nerd fonts. I've been using the regular one-line output on my script, but as I mostly use Nerd Fonts would be cool to have that support as well. Thanks for the great software nonetheless.

**Comment by chubin** [comment]
Accepted. We will do it. And please feel free to contribute

**Comment by nathanielevan** [comment]
Hi, and thanks for the great software! Really appreciate your work on wttr.in. However, using both `v2d.wttr.in` and `v2n.wttr.in`, I notice that while the precipitation graph already uses Nerd Fonts, the `Weather: ` line doesn't seem to (it uses an emoji character instead):

![scshot-040921-121757](https://user-images.githubusercontent.com/9361126/132083578-cc0c2d9c-5ff7-44aa-9d2c-03d828e26f2c.png)

Is this a work-in-progress, or does it happen to be an oversight? Additionally, how may I help?

**Comment by chubin** [comment]
@nathanielevan Nathaniel thank you it is a good catch. Will be fixed

**Comment by Maritsu** [comment]
any updates on one-line nerd font support?
